src/clj/game/cards.clj

```diff
@@ -207,6 +207,24 @@
                                                (vals selected-cards)))]
                            (effect-completed state side (make-result eid {:number counter-count :msg msg})))))}))
 
+(defn autoresolve-toggler [toggle-kw ability-name]
+  "Makes a card ability which lets the user toggle auto-resolve on an ability. Setting is stored under [:special toggle-kw]."
+  {:label (str "Toggle auto-resolve on " ability-name)
+   :prompt (str "Set auto-resolve on " ability-name " to:")
+   :choices ["Always" "Never" "Ask"]
+   :effect (effect (update! (update-in card [:special toggle-kw] (fn [x] (keyword (clojure.string/lower-case target)))))
```

Comment:
```suggestion
   :effect (effect (update! (update-in card [:special toggle-kw] (fn [x] (slugify target))))
```
This only works if you require slugify from `jinteki.utils`, but it's there to be used!